cdc/sink/sink.go

```diff
@@ -118,3 +118,29 @@ func NewSink(ctx context.Context, changefeedID model.ChangeFeedID, sinkURIStr st
 	}
 	return nil, cerror.ErrSinkURIInvalid.GenWithStack("the sink scheme (%s) is not supported", sinkURI.Scheme)
 }
+
+// Validate the health of the sink
```

Comment:
In fact, what we are checking is whether sinkuri and config are valid.
So this comment may be better?  
```
Validate whether the sinkuri and config are valid.
```